hacks/update-jenkins-plugins/plugins.txt

```diff
@@ -0,0 +1,54 @@
+ace-editor:1.1
+branch-api:2.0.9
+cloudbees-folder:6.0.4
+credentials:2.1.13
+display-url-api:2.0
+durable-task:1.13
+git-client:2.4.5
+git-server:1.7
+git:3.3.0
+handlebars:1.1.1
+icon-shim:2.0.3
+jquery-detached:1.2.1
+junit:1.20
+mailer:1.20
+mapdb-api:1.0.9.0
+matrix-auth:1.5
+matrix-project:1.10
+mercurial:1.59
+momentjs:1.1.1
+multiple-scms:0.6
+pipeline-build-step:2.1
+pipeline-graph-analysis:1.3
+pipeline-input-step:2.7
+pipeline-rest-api:2.6
+pipeline-stage-step:2.2
+pipeline-stage-view:2.6
+pipeline-utility-steps:1.3.0
+plain-credentials:1.4
+scm-api:2.1.1
+script-security:1.27
+ssh-credentials:1.13
+structs:1.6
+subversion:2.7.2
+workflow-aggregator:2.1
+workflow-api:2.13
+workflow-basic-steps:2.4
+workflow-cps-global-lib:2.8
+workflow-cps:2.30
+workflow-durable-task-step:2.11
+workflow-job:2.10
+workflow-multibranch:2.14
+workflow-remote-loader:1.4
+workflow-scm-step:2.4
+workflow-step-api:2.9
+workflow-support:2.14
+
+# kubernetes version 0.11 has been used successfully with jenkins-1, so
+# override its encoded jenkins version requirement (2.32.1).
+kubernetes:0.11;jenkins-version-override
+
+openshift-login:0.12
```

Comment:
Conversely, I think the next two should specify specific versions, and not leverage "latest".